buildkite-cli/plan.sh

```diff
@@ -0,0 +1,53 @@
+pkg_name=buildkite-cli
+pkg_origin=core
+pkg_version="0.3.0"
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=("MIT")
```

Comment:
I don't see any licensing information in their repo,  however their other software is licensed MIT (including their [homebrew tap](https://github.com/buildkite/homebrew-cli) for this package) so this is most likely correct.

Reply:
Okay, formally asked here: https://github.com/buildkite/cli/issues/33